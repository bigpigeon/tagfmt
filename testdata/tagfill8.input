//tagfmt -f "json=hungary(:field)+:tag_extra"

package main

type OrderDetail struct {
    ID       string   `json:",omitempty"`
    UserName string   `json:",omitempty"`
    OrderID  string   `json:",omitempty"`
    Callback string   ``
    Address  []string ``
}
