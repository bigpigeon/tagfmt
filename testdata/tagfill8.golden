//tagfmt -f "json=hungary(:field)+:tag_extra"

package main

type OrderDetail struct {
	ID       string   `json:"id,omitempty"`
	UserName string   `json:"user_name,omitempty"`
	OrderID  string   `json:"order_id,omitempty"`
	Callback string   `json:"callback"`
	Address  []string `json:"address"`
}
